apps/ledger-live-desktop/src/mocks/init.ts

```diff
@@ -3,9 +3,22 @@ async function enableMocking() {
 
   if (mswEnabled) {
     try {
-      const { startWorker } = await import("~/mocks/browser");
+      const { startWorker, toggleSimulate500Error } = await import("~/mocks/browser");
 
       await startWorker();
+
+      // Expose toggle function globally for debugging
+      if (__DEV__) {
```

Comment:
The project setup means ENABLE_MSW implies dev mode is enabled (`"start:msw": "cross-env NODE_ENV=development ENABLE_MSW=true node ./tools/main.js",`)

Maybe it would make more sense if line 4 was `if (mswEnabled && __DEV__) { ...`?